packages/shared/src/UI/components/SocialAccountList/SocialTooltip.tsx

```diff
@@ -0,0 +1,49 @@
+import { useSharedI18N } from '@masknet/shared'
+import { NextIDPlatform } from '@masknet/shared-base'
+import { makeStyles, ShadowRootTooltip } from '@masknet/theme'
+import { resolveNextIDPlatformName } from '@masknet/web3-shared-base'
+import { Typography } from '@mui/material'
+import type { PropsWithChildren } from 'react'
+
+interface StyleProps {
+    isMenuScroll?: boolean
+    hidden?: boolean
+}
+
+const useStyles = makeStyles<StyleProps>()((theme, { hidden = false }) => {
+    return {
+        tooltip: {
+            visibility: hidden ? 'hidden' : 'visible',
+            backgroundColor: theme.palette.maskColor.publicMain,
+            color: theme.palette.maskColor.white,
+        },
+        arrow: {
+            color: theme.palette.maskColor.publicMain,
+        },
+    }
+})
+interface SocialTooltipProps {
+    platform?: NextIDPlatform
+    hidden?: boolean
+}
+export function SocialTooltip({
+    children,
+    hidden = false,
+    platform = NextIDPlatform.NextID,
+}: PropsWithChildren<SocialTooltipProps>) {
+    const { classes } = useStyles({ hidden })
+    const t = useSharedI18N()
+    return (
+        <ShadowRootTooltip
+            classes={{ tooltip: classes.tooltip, arrow: classes.arrow }}
+            arrow
+            placement="top"
+            title={
+                <Typography style={{ padding: '6px 2px', whiteSpace: 'nowrap' }} fontSize={14}>
```

Comment:
The default font-size is `14px`.
```suggestion
                <Typography style={{ padding: '6px 2px', whiteSpace: 'nowrap' }}>
```